db/controllers/controller.js

```diff
@@ -15,7 +16,6 @@ exports.getArticlesById = (req, res, next) => {
   fetchArticlesbyId(article_id).then((article) => {
     res.status(200).send({ articles: article });
```

Comment:
Sorry to go back to a previous ticket - this endpoint should be sending information on the key of `article` as it is one article coming back